### DISCUSSION TITLE
Who's hiring? (Nov 2021)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#31542] [Ideas]
# What is this?
This discussion is for companies looking for people with Next.js experience. Both permanent positions and contract work are welcome.

# Companies
Please use the following format followed by a brief description of the role:

> COMPANY_NAME | ROLE | LOCATION | Full-time | $100-150k base | LINK

Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

## Sample Job Post

Here's a good example job post.

> Gusto | Software Engineer | SF, NYC, Denver, or Remote (US/Canada) | Full-time | $143-$243k base | [Link](https://boards.greenhouse.io/gusto/jobs/3441405)
> 
> Gusto's hiring in just about every engineering department, but the above link is for our product infrastructure engineering team (the team I'm on). Gusto provides payroll, benefits, and HR tools to small businesses in the United States. We recently raised a Series E and have the privilege of serving over 200K small businesses.
> 
> I've been at the company for 5 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me directly.

---

If you're a candidate looking to post your information, check out #31541.

**Comment by jordie23** [comment]
CarExpert.com.au | Full-stack Developer | Remote (Australia only), or Sydney, Melbourne or Brisbane | Full-time | AUD$110-140k base | [Link](https://www.seek.com.au/job/54991997)

[CarExpert](https://www.carexpert.com.au/) is a fast growing startup and we're looking for more developers to help us keep development up with our ideas. Our mission is to create the world's most trusted and enjoyable journey to new car ownership, so we are working on innovative solutions that will make researching new cars a fun and stress-free experience.

I'm the CTO. We have a team consisting of myself, 3 full-time devs & 2 contractors. We're looking for another 2 full-time full-stack developers. Daily work will vary between NextJS and our AWS backend. We're currently implementing new parts of our site using KeystoneJS CMS. 

At the moment all our developers are working remotely in various locations in Australia. You're welcome to WFH full time, or come into one of our offices if you live in Syd, Melb or Bris. You must have the right to work in Australia. Please use the seek link above for more info and to apply.

**Comment by garmeeh** [comment]
Pluto Digital | Frontend Developer | Remote (ideally Ireland/UK) | Full Time | [Job Spec](https://plutodigital.com/jobs/react-js-front-end-developer/)

[Pluto Digital](https://plutodigital.com/) is a leader in Decentralised Finance (DeFi) staking and yield farming software platforms, applications and services. Our mission is to make the world of Crypto and Decentralised Finance (DeFi) more accessible to mainstream investors and consumers.

I recently joined Pluto Digital to help head up the frontend effort and we're currently looking for a second frontend developer to join the team. You will be mainly working on Next.js dapps using the latest Web3 technologies. With this being a start up, the team is only newly formed so it is an exciting time to join. (a [well funded](https://plutodigital.medium.com/pluto-blasts-beyond-moon-to-50m-within-50-days-of-inception-ccb090600d93) start up) Check out the Job Spec link above for more information and how to apply.

**Comment by dan-made** [comment]
Made In | Front-End Engineer | Austin, Remote (Continental US) | Full-time | [Link](https://airtable.com/shrR36QLvogOyLeJq)

Backed by some of the best chefs in the world, [Made In](https://madeincookware.com) is the leader in the digitally-native kitchen space. We bring Chef expertise and centuries-old cookware manufacturing techniques to craft professional-quality kitchen tools for the home and professional cook.

Made In engineering is committed to building a best-in-class eCommerce experience. The Front-End Engineer will be working with the product team and fellow engineers to design, implement, and ship user-facing features.

Do you like using Next? Deploying to Vercel? Styling with Tailwind? Are you obsessed with accessibility? Are you excited about the potential of middleware and React Server Components? Do you have hot takes about design systems? Have you read the [Stories RFC](https://github.com/vercel/next.js/discussions/30628)? If you said yes to any of these, we would love to hear from you.

**Required qualifications:**

* 3+ years of front-end development.
* Experience writing code in modern JavaScript frameworks.
* Ability to think holistically but with an eye towards details.
* Appetite for collaboration.
* Ability to communicate effectively on complex topics with a non-engineering audience.
* Interest and ability to thrive in a fast-paced startup environment – wear multiple hats, make decisions, and ride out context-switching.

**Full-Time Benefits**

* Remote eligible
* Competitive salary
* Equity grant
* Healthcare, Dental, and Vision
* Flexible vacation policy
* Generous employee discount

If this sounds like the role for you, [click here to apply](https://airtable.com/shrR36QLvogOyLeJq).

Made In headquarters is based in Austin, TX. This position is remote eligible within the continental United States.

Must be eligible to work in the United States. We are unable to provide Visa sponsorship for employment at this time.

Made In is an equal opportunity employer. We celebrate diversity and are committed to creating an inclusive environment for all employees.

**Comment by edouardb** [comment]
Koyeb | Senior Frontend Engineer | Remote (Europe) | Full-time | [Link](https://www.koyeb.com/careers)

[Koyeb](https://www.koyeb.com) is a developer-friendly serverless platform to deploy apps globally. No-ops, servers, or infrastructure management.

We are looking for a passionate Senior Frontend Engineer who loves Cloud Infrastructure Platforms. If you want to build customer-facing products which people rely on every day, this is the right job for you.

We’re remote-friendly and have staff based in Paris and Lyon, France, check it out!

**Comment by aegiz** [comment]
Circularise | Frontend Developer | Remote (ideally The Netherlands or EU) | Full Time | [Job Spec](https://www.circularise.com/careers/mid-level-senior-frontend-developer)

[Circularise](https://www.circularise.com) is a supply chain traceability start-up founded in The Netherlands in 2016. It’s our mission to bring traceability to global supply chains and to accelerate the world’s transition to a Circular Economy. 

**Requirements**
• You have at least 3-5 years of experience as a Frontend developer / Integration engineer.
• You know React and Typescript and can write robust and maintainable JavaScript code using modern standards (ES6+).
• You know the best practices (responsive design, architectural pattern that scales, etc.).
• You have some strong CSS skills, focused on writing maintainable code.
• You have a good knowledge of Git and how to work collaboratively.
• You are a great communicator (both written and verbal) and like to share knowledge with the rest of the team.
• You are proactive, adaptable and can work independently when needed.

**Scope of work**
• Work towards a production ready system for a large number of clients.
• Ensuring feasibility of UI/UX designs and modifying specifications of complex applications.
• A transparent industry protocol while still maintaining strong privacy and security features (for us, but also our clients).
• Make use of exciting new tech and their caveats: decentralisation, public blockchains, Zero Knowledge Proof (and at a later stage big data analytics).
• Meeting demands and requirements of various industry clients (e.g. audits).

**Benefits**
• A competitive salary
• Flat hierarchy
• 25 days of paid time off
• Travel allowance
• Semi-remote, work from home policy
• Flexible working hours
• Annual company holiday
• Be part of a new industry standard

**Comment by allang** [comment]
Idea Sans | Software Engineer | Remote | Freelance | $Negotiable | [Email](mailto:allan@ideasans.com)

**Scope of work**
In need of a freelance engineer to do ongoing work on a NextJS app. The product has a small scope and codebase is not yet complex. I'm in need of help closing some tasks on a week-to-week basis. You'll be working with me on a stack that looks like: **NextJS**, **Mongodb**, and **Tailwind**. Experience with **NextAuth** and **Stripe** is a big benefit.

**Contact**
If interested, please email with your level of experience, your GitHub username, whether you've worked with NextAuth or Stripe, and your hourly rate.

**Reply by erinoggz** [reply]
I'm open for this role. Is it still available?

**Reply by volodymyr0507** [reply]
As I am a MERN stack developer, I am very interested in your job.
I have rich experience in Next, React, Node, MongoDB and Tailwind CSS.
Especially, I have experience in NextAuth and Stripe.
My github name is talentedev
hourly:30usd/hr
Looking forward to hearing  form you.
thanks.

**Reply by himanshustackera** [reply]
Is it still open?

**Comment by hect1c** [comment]
Palqee Technologies | Senior Frontend Developer | Remote (UK or Brazil) | Full Time | [Job Spec](https://www.palqee.com/careers/)

At [Palqee](https://www.palqee.com/), we are changing the way companies operationalise Privacy, Data Protection and Data Governance, by building intuitive technologies that give peace-of-mind about your most important data. Our mission is to create trusted relationships between businesses and people around the world.
With a passion to solve global privacy concerns, our team is building the most advanced Privacy Management ecosystem yet.

## RESPONSIBILITIES

- Provide engineering leadership with a focus on user interface design and development
- Design and implement user interfaces in collaboration with team of full stack engineers
- Demonstrate a commitment to Palqee core values
- Provide feedback and implementation advice to team members
- Experience working in agile environments that have embraced DevOps and Continuous Delivery best practices
- Developing automated tests, and deployment processes
- Create testable requirements and write and execute unit tests
- Plan and estimate time efforts and work with team and CTO
- Insistent on recognising systems imperfections and replace mistakes with rapid improvement
- Provide status updates during daily stand-up meetings
- This role requires that you are a self-starter with the ability to deliver on time with quality
- Performs other duties as assigned

## REQUIRED EXPERIENCE

- 5+ years of related professional experience in a structured, process-driven environment
- 4+ years of experience coding HTML, CSS, Javascript and other UI technologies with a portfolio of your work
- 4+ years of experience developing in React
- 2+ years of experience working with GraphQL
- Experience with a variety of JavaScript frameworks and libraries such as: Apollo Client, NextJS, Typescript and testing with Jest, Cypress, and Chai
- Experience with Agile development process and lean methodology
- Experience with GIT and workflows (Gitlab)
- Experience implementing web applications from a provided list of requirements and design mock-ups
- Outstanding verbal and written communication in English
- Great interpersonal skills - experience working with, mentoring and collaborating with international team members
- Must be responsive, detail oriented and have a strong sense of appropriate planning for meeting deadlines
- Strong sense of ownership and accountability
- Experience in bringing multiple viewport responsive and adaptive web designs to life
- Awareness and practice of UI performance best practices
- Able to bridge communication between technical and non-technical stakeholders and customers
- Enjoy working in a cross-functional team
- Excellent autonomous problem-solving ability
- Work efficiently in an environment where not everything is organized and structured
- No ego, we’re a flat hierarchy business. We respect each others’ experience and profession but are open to other people’s ideas and input

## WHAT WE OFFER

- Be a part of a fast-growing technology startup and shape the future of privacy and data protection;
- £48,000 - £60,000/yr. salary based on experience;
- Stock Options;
- 25 days of annual leave plus your local national holidays;
- Family friendly work conditions and environment;
- Remote work + Flexi-hours;
- Opportunity to relocate to one of our global offices;
- Annual Palqee Employees Summit;
- Monthly get togethers;
- Tickets for the best conferences in the industry;
- Budget for training, material and development;
- Opportunity to enrich our Spotify playlist.

**Comment by DrianHillman** [comment]
**SharpLink Gaming | Front-End Developer | Remote/Canton,CT | Full-time | $85k-$120k base | [Job Listing](https://www.sharplink.com/careers/front-end-developer-react-free-to-play/)**

My team at SharpLink is looking to add a few Front-End Developers. We use Next.js to build free-to-play fantasy games for major sports leagues and media partners. Check out the link above to view the job listing & apply.

→ **Update: Roles were filled as of Jan '22**

**Reply by volodymyr0507** [reply]
Dear DrianHillman
As I am a senior frontend developer with  6 +years of experience in React, Vue, Tailwind CSS, BootStrap and HTML CSS, I am very interested in your job.

my email:topdev124@gmail.com
discord: Goal Archiver#7943
Looking forward to hearing from you.

**Comment by jojopheenie** [comment]
Forum Brands| Full Stack Engineer | NYC | Full-time | TBD based on level | [LINK](https://www.forumbrands.com/careers/ )

Hey, all! My company is hiring full stack engineers of all levels! You’ll be working with AWS micro-service architecture, Node, GraphQL, React, Apollo, NextJS, Docker, Serverless, PostreSQL, Jest and you'll have lots of exposure to ML/data science. Amazing, supportive, inclusive, and collaborative team in the brand buying and scaling sphere with tons of training for career growth. Currently remote but will eventually go back to the office in NYC at least 3 days a week. Open to being fully remote for the right people. Thanks!

**Comment by braden-clerk** [comment]
Clerk | Senior Frontend Engineer | Remote | Full-time | $100-150k base | [Job Description](https://clerk.dev/blog/hiring-frontend-full-stack-components)


**[Clerk](https://clerk.dev) is hiring a senior frontend team to expand our full-stack component library**

Seeking experts in CSS, browser APIs, JS bundling, React, Next, other modern Web frameworks, react-native, Swift, Kotlin and Developer Experience

We're looking for developers who can help improve and grow our full-stack component library as Clerk expands into Organization Management. We're seeking expertise across many frontend disciplines!

**Reply by softIntelligent** [reply]
As I front end developer, I am interested in this opportunity.
My discord user name is : _Lee#0561
Hope your reply.
Thank you

**Comment by fabien-huet** [comment]
[Chantelle](https://chantelle.com/) | senior + mid front end developers | full remote | 55-70k (see details in the package) | Europe

Chantelle is the only lingerie company that managed to keep design and manufacturing in France.

### The context

We have old e-commerce websites running on Magento and we switch them to a headless e-commerce platform (centra or bigcommerce, TBD). We need to build the front ends for those websites. The project has not completely started yet, so it would be pretty much from scratch. We need to add a senior and a mid developer to the team.

### Tech / skills that you will use

- TypeScript and NextJS/React
- GraphQL (and some REST for third party providers)
- styled components, HTML / CSS
- English (you must have good written and verbal communication skills)

### Of course

Code quality best practices, accessibility, security, performances, testing, SEO... general dev skills.
Kindness, cheerfulness, work ethic, honesty... general human skills.

### Our recruitment process:

- Send your resume / github / linkedin to fabien.huet@groupechantelle.com => screening
- Technical Interview in pair programming ~ 1 hour
- Cultural Fit Interview ~ 1 hour
- Offer presentation / Contract ~ 1 hour

If you're outside of the EU we can sponsor your VISA and offer a relocation package.

### Package

- Full remote possible, but you can come to our WeWork office in Paris and Berlin (or any WeWork office in the world)
- Flexible work hours outside of our core hours (10 to 16 CET)
- A dedicated yearly budget for your training and conferences with up to 10 dedicated days
- 25 paid leaves with extra holidays on the 24th and 31st of December
- Quarterly team retreats
- Evolution mid > senior > lead over the years.

Since we offer German or French permanent contracts the growth salaries differ, but end with the same employer cost and employee net salaries.
- for the mid (degree / training + min 2-5 years in the industry) : 55k brutto in Germany, 48k brut in France
- for the senior (degree / training + min 5-8 years in the industry) : 70k brutto in germany, 60k brut in France

**Comment by crypt096** [comment]
[HIRING] [100% REMOTE] [SEVERAL POSITIONS] [FULL-TIME] 

Hey guys, I am looking for following roles: 

- DevOps/ Cloud Engineer - with Azure or AWS - mid and senior level
- Android Developers - mid and senior level 
- iOS Developers - mid and senior level 
- .NET Developers - mid and senior level 
- JavaScript with with Vue / Angular / React / React Native or Node - mid and senior level 
- Automation QA - mid and senior level

**VERY IMPORTANT: Able to work in GMT+0 - GMT+2 timezones and has +2.5yrs of experience **

If you find yourself compatible for any of those positions feel free to contact me in DM **( I will need your resume in PDF format, email and your phone number)**.
Jobs are 100% remote with very good compensation.

**Comment by alxheller** [comment]
**[Parity Technologies](https://www.parity.io/) | Frontend Web Developer | Remote (Central European Timezone) | Full-time | [Link](https://boards.greenhouse.io/parity/jobs/4805058003)**

Parity develops some of the leading open-source blockchain projects, including Polkadot, Kusama, and Substrate. 

Parity is hiring a wide range of frontend expertise from back-of-the-frontend to front-of-the-frontend, so please apply if you have frontend experience that could add to the team. 

I've been at the company for 3.5 years and really enjoy how exciting the company is. It's growing quickly and is a fun place to be! Feel free to apply using the link above. Lmk if you have any questions! 😀

**Comment by digitalbase** [comment]
Prezly | Software Engineer | Remote (we're in europe) | Full-time | $70-$140 base | Link

We are looking for a frontend engineer with deep understanding of Frontend/Web technologies (NextJs, typescript, scss, styling) and best practices when it comes to frontend/web development. You should have senior level experience (~5 years) building websites, with at least 1 year of that experience using nextjs. You will work on a variety of projects, mostly on the main website and the newsrooms of our customers. Your work will ship continuously so you will have a direct impact in our customers’ experience and the overall trajectory of the business.

Apply through https://careers.prezly.com/o/webfrontend-developer-typescript-nextjs-css

**Comment by sourceresslana** [comment]
Solutions By Design (SBD) | Lead UI Developer | USA - Remote | Full-time | 155 to 175K (negotiable) | [React Developer](https://sbd2.applicantpro.com/jobs/2171403.html)

Are you a tech-savvy professional curious about new digital technologies and aspire to combine usability with visual design? If you're interested in creating a user-friendly environment by writing code and moving forward in your career, let's chat!

The React Developer will be part of the team that builds the 'client-side' of our new web-based applications for our Federal Health IT client. This role will translate our company and customer needs into functional and appealing interactive applications. You will build out functional user interfaces through close collaboration with back-end developers to integrate with the server-side application logic and UI/UX web designers to create a functional and attractive digital environment with improved usability.

**Comment by reallymemorable** [comment]
Stealth | Senior Eng | Remote | Full-time | $150k base

We're hiring a Senior Software Engineer at our stealth consumer app that aims to replace traditional search and advertising for a large segment of the economy. The team comprises serial entrepreneurs and an ex-Capital One executive who worked directly with the CEO to build that from a tiny company to a major consumer finance brand.

As a member of our engineering team, you'll help design, implement, and maintain new features on our flagship product. 

TC: $200k-230k + equity ($150k base + $50k-80k bonus + equity)
Fully Remote

Tech Stack:
- React + Typescript
- Nextjs
- Node
- Prisma
- PostgreSQL

Qualifications:
- Experienced in full-stack development
- Capable of debugging frontend & backend issues
- Capable of implementing features that span the frontend & backend
- Comfortable working at all levels of the stack

Please send resumes to applyfortherolehere@gmail.com

**Comment by wittwerch** [comment]
[Digital Innovation Lab](https://diginlab.com/) |  Full Stack Developer | Remote | Freelance | $Negotiable | [Email](mailto:jobs@diginlab.com)

We are currently looking for an experienced Full Stack Developer who is ready to join the Digital Innovation Lab team. The Digital Innovation Lab develops digital products for startups and innovative corporates. We help getting MVPs to the market and shape them to create successful companies.

Why you’re the perfect fit:

- in depth knowledge of TypeScript
- profound knowledge of Node.js and MVC frameworks (preferably NestJS)
- worked with React and ideally Next.js :)
- experience with PostgreSQL
- open minded and you’ll love to discuss architecture matters in a team
- you have worked with Gitlab and Google Cloud Platform

What do we bring to the table:

- Work in the startup scene and help to bootstrap the next unicorn
- Responsibility from day one
- Modern software engineering culture
- Work from anywhere
- Opportunity to meet the team in Switzerland

I'm the co-founder and CTO of the Digital Innovation Lab. We're a small team after all but if you're looking to make an impact and build innovative digital products, feel free to reach out to me!

**Reply by dan-dervich** [reply]
hey, i work alot with nodejs I've dabbled in nest but i'd love to apply and and prove my use. If you'd like to test me on nest or whatever language yoy want I'd be more than willing to hop on a zoom meeting and you can test me on there.

**Comment by MichaelCastelbuono** [comment]
[Research & Design](https://rd.digital/) | Contract Developer | Remote | Part-time | [LINK](https://rd.homerun.co/senior-javascript-developer/en)

We am currently looking for a US/Canada-based contractor for a 4-6 month project with the potential to extend beyond its original time frame. Our team is rebuilding an online donation platform using Next.js. [Apply](https://rd.homerun.co/senior-javascript-developer/en) if you are interested.

**Comment by telepathicbeing** [comment]
Hi, I am a research fellow, who can pay at hourly rates (based on your experience and level),  to fix small UI issues and help with general maintanence in an existing next.js art/heritage project.  Please contact cobi.vantonder@york.ac.uk to find out more.  Project here:  acousticatlas.de

**Comment by pratik-dani** [comment]
ClientList | NextJS Intern | India Remote | Internship | $Negotiable | [Link](https://www.linkedin.com/jobs/view/3498530376/)

ClientList is looking for a Intern who can help us build the frontend. The intern will be closely working with me on the development.

I am co-founder and CEO of the company, I have been working on this for 2.5 years now. Feel free to apply using the link above or reach out to me directly.
